gp-includes/routes/translation.php

```diff
@@ -489,7 +489,7 @@ private function _bulk_approve( $bulk ) {
 					case 'changesrequested':
 						$message = sprintf(
 						/* translators: %s: Translations count. */
-							_n( 'Error requesting changes in %s translation.', 'Error requesting changes in %s translations.', $error, 'glotpress' ),
+							_n( 'Error with requesting changes in %s translation.', 'Error with requesting changes in %s translations.', $error, 'glotpress' ),
```

Comment:
I am not an English native speaker, but I think the current text is fine. Can a native English speaker review this?

Reply:
The issue here isn't only being correct, is that it's not in line with the other sentences.

See the current strings for these errors:
![imagem](https://user-images.githubusercontent.com/7371591/232131959-ae70588e-94dc-4eba-8c78-4ef68d94d8e1.png)

Reply:
Also you can see in the code that all of them are used twice, and this about the status "requesting changes" have two very similar texts, it's usefull to combine in one.